src/extension/background-script/connectors/nwc.ts

```diff
@@ -215,6 +223,15 @@ class NWCConnector implements Connector {
   connectPeer(args: ConnectPeerArgs): Promise<ConnectPeerResponse> {
     throw new Error("Method not implemented.");
   }
+
+  private convertCustomRecords(
+    customRecords: Record<string, string>
+  ): TlvRecord[] {
+    return Object.entries(customRecords).map(([key, value]) => ({
+      type: parseInt(key),
+      value: Hex.parse(value).toString(Base64),
```

Comment:
This is completely wrong. Do you know the format of WebLN keysend and NIP-47 keysend?